### DISCUSSION TITLE
Refresh token auto inactive or delete

### DISCUSSION THREAD

**Discussion by henrypham299** [ory/hydra#2646] [Q&A] [answered]
**Describe the bug**

I dont know how to reproduce that bug, because this bug occur sometime

I use `AppAuth` to login and build the mechanism to refresh the token, but sometimes i cant not refresh token I checking on the hydra data i saw the `refresh_token` has is_active = false, or maybe deleted I cant check it. I dont know any reason for this issue, could you help me to check it? Is it relate to 1 account login on multiple devices?

_Server logs_

```
{"debug":"The refresh token has not been found: not_found","message":"invalid_grant","reason":"","status":"Bad Request","status_code":400}
```

**Expected behavior**

Need to refresh token normally

**Environment**
- Version: v1.10
- Environment: Docker

Could you guys tell me, do we have another way to de-active or delete the refresh token except revoke the accessToken?

**Comment by aeneasr** [comment, marked as answer]
You’re probably using the token twice, make sure to not accidentally send it twice!

**Reply by henrypham299** [reply]
@aeneasr 
But when using the token twice, i get the message below?

```
{
    "error": "token_inactive",
    "error_description": "Token is inactive because it is malformed, expired or otherwise invalid. Token validation failed."
}
```

**Comment by ahmedbally** [comment]
old refresh_tokens expired though and give same error 
is should be have only one refresh_token valid ?

**Comment by vinckr** [comment]
Hey @henrypham299 , just checking in if your problem still persists. 
I will mark this as answered for now, but feel free to come back to the discussion any time.
Thanks a ton :bee:
